src/app/app.ts

```diff
@@ -33,7 +34,18 @@ export interface IAppConfig {
 export default function(config: IAppConfig) {
   const app = express();
 
-  app.use(pinoMiddleware({logger: config.logger}));
+  app.use(pinoMiddleware({
+    logger: config.logger,
+    serializers: {
+      req: (req: IncomingMessage) => ({
+        method: req.method,
+        url: req.url,
+      }),
+      res: /* istanbul ignore next */ (res: ServerResponse) => ({
```

Comment:
I think coverage could be caught for this using the "main" tests, but I don't feel strongly enough to request a change for this.

Reply:
It felt bad when I put that in... It's a simple check, yet feels unnecessary...